src/components/artistas/ArtistaForm.jsx

```diff
@@ -66,7 +69,18 @@ class ArtistaForm extends Component {
   }
 
   addArtista() {
-    const { name, active, country, gallery, galleryUrl, bioRawContent, cvRawContent, images, video } = this.state;
+    const {
```

Comment:
this is def a better format for these things

Reply:
ikr?! its better when it's more than 3